docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management-Transition.rst

```diff
@@ -0,0 +1,47 @@
+Object Lifecycle Management: Transition
+=======================================
+
+**Prerequisite:** You must have established a bucket with at least one object
```

Comment:
There doesn't necessarily have to be an object in the source bucket to set a rule.

Reply:
fixed